app/components/file-details/dynamic-scan/drawer-old/index.ts

```
@@ -0,0 +1,174 @@
+      this.args.onClose();
+
+      file.setBootingStatus();
+
+      this.args.pollDynamicStatus();
+
+      this.notify.success(this.tStartingScan);
+    } catch (error) {
+      const err = error as AdapterError;
+      let errMsg = this.tPleaseTryAgain;
+
+      if (err.errors && err.errors.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)